### PR TITLE
Feature/composer install

### PR DESCRIPTION
**Pull request by jrfoell**
No issue to close here, just wanted you guys to see what my approach to all the plugin requirements would look like. It's a biggie b/c I deleted all of the plugins checked into your repo. _Actual_ changes are found in the `composer.json` and `.gitignore` files.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,18 +42,33 @@
...
+    "wpackagist-plugin/gutenberg": "^9.8",
+    "wpackagist-plugin/query-monitor": "^3.5",
+    "wpackagist-plugin/resmushit-image-optimizer": "^0.3",
+    "wpackagist-plugin/stream": "^3.4",
+    "wpackagist-plugin/wp-graphql": "^1.1",
+    "wpackagist-plugin/wp-search-with-algolia": "^1.6",
     "webdevstudios/advanced-custom-fields-pro": "^5.9",
     "webdevstudios/gravityforms": "^2.4",
     "webdevstudios/mu-autoload": "^1.0",
-    "webdevstudios/sso": "^2.0",
```
Comment (oddevan) [inline composer.json:40]: Does this remove `webdevstudios/sso` from the project? Do we need it?

File: composer.json [hunk 2]
```diff
@@ -34,18 +42,33 @@
   },
   "require": {
     "pristas-peter/wp-graphql-gutenberg": "^0.3.7",
+    "wp-graphql/wp-graphql-acf": "^0.4.0",
+    "wp-media/wp-rocket": "^3.6",
+    "wpackagist-plugin/add-wpgraphql-seo": "^4.11",
+    "wpackagist-plugin/block-manager": "^1.1",
+    "wpackagist-plugin/cookie-notice": "^1.3",
```
Comment (gregrickaby) [inline composer.json]: Don't need

File: composer.json [hunk 3]
```diff
@@ -34,18 +42,33 @@
   },
   "require": {
     "pristas-peter/wp-graphql-gutenberg": "^0.3.7",
+    "wp-graphql/wp-graphql-acf": "^0.4.0",
+    "wp-media/wp-rocket": "^3.6",
```
Comment (gregrickaby) [inline composer.json]: Don't need

**Comment by jrfoell** [comment]
@oddevan `webdevstudios/sso` isn't needed because requiring the WDS add-on `webdevstudios/sso-addon` brings in the _correct_ companion SSO version that goes with it.

File: composer.json [hunk 4]
```diff
@@ -34,18 +42,28 @@
...
     "webdevstudios/advanced-custom-fields-pro": "^5.9",
     "webdevstudios/gravityforms": "^2.4",
     "webdevstudios/mu-autoload": "^1.0",
-    "webdevstudios/sso": "^2.0",
     "webdevstudios/sso-addon": "^1.0",
     "webdevstudios/wds-blocks": "^2.0",
     "webdevstudios/wp-migrate-db-pro": "^1.9",
     "webdevstudios/wp-migrate-db-pro-cli": "^1.3",
     "webdevstudios/wp-migrate-db-pro-media-files": "^1.4",
-    "wp-graphql/wp-graphql-acf": "^0.4.0",
```
Comment (gregrickaby) [inline composer.json:46]: @jrfoell We need `wp-graphql-acf`, can you please add it back in.

**Review by gregrickaby — CHANGES_REQUESTED** [review]

**Comment by jrfoell** [comment]
LOL, okay

wp-graphql-acf-dont-need

I added `lazy-blocks` via composer as well

**Pull request merged**